### DISCUSSION TITLE
Ein DTU, mehrere Wechselrichrter

### DISCUSSION THREAD

**Discussion by DrBratwurst** [hoylabs/OpenDTU-OnBattery#990] [Ideas]
Hallo, liebe Gemeinde,

ich nutze nun seit 2 Monaten einen Open DTU on Battery und einen HM 400 in harmonischer Symbiose ;-) Nun war mein Gedanke, dass es doch sicherlich ganz einfach sein muss, die angeforderte Leistung (vom Stromzähler) durch die Anzahl zu steuernder Wechselrichter zu teilen und diesen Wert an die teilnehmenden Wechselrichter zu senden.
Leider musste ich feststellen, dass dies so aktuell gar nicht implementiert bzw. angedacht ist. Hat das einen technischen Hintergrund und ist so nicht umsetzbar oder wurde dieser Gedanke schlichtweg nicht gedacht?
Hintergrund des ganzen bei mir persönlich ist, dass ich gerne 3x 400 Watt HM für je eine Phase verwenden möchte und die benötigte Leistung auf alle drei Phasen gleichmäßig verteilen möchte.

Vielleicht bin ich ja mit meiner Idee nicht ganz alleine und es lässt sich mein Wunsch doch mit drei Mausklicks implementieren. Wenn nicht, hat ja eventuell jemand eine Idee für einen ganz anderen Ansatz...

Beste Grüße an die Community und an all die tollen Entwickler da Draußen :-)

**Comment by Manos1966** [comment]
> und es lässt sich mein Wunsch doch mit drei Mausklicks implementieren

Es lässt sich nicht mit drei Mausklicks implementieren.

Ein Ansatz existiert aber doch: 
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/FAQ#can-opendtu-onbattery-dynamic-power-limiter-control-more-than-one-inverter

**Reply by DrBratwurst** [reply]
Vielen Dank für deinen Hinweis und den entsprechenden Link. Ich werde die Sache vorerst mit Home Assistant und eine Automation totschlagen und mir den kleinsten Hoymiles für drei Phasen besorgen. Dann sollte es für mich passen.

Beste Grüße
